fmriprep/interfaces/resampling.py

```diff
@@ -0,0 +1,641 @@
+"""Interfaces for resampling images in a single shot"""
+
+import asyncio
+import os
+from functools import partial
+
+import nibabel as nb
+import nitransforms as nt
+import numpy as np
+from nipype.interfaces.base import (
+    File,
+    InputMultiObject,
+    SimpleInterface,
+    TraitedSpec,
+    traits,
+)
+from nipype.utils.filemanip import fname_presuffix
+from scipy import ndimage as ndi
+from scipy.sparse import hstack as sparse_hstack
+from sdcflows.transform import grid_bspline_weights
+from sdcflows.utils.tools import ensure_positive_cosines
+
+from ..utils.asynctools import worker
+from ..utils.transforms import load_transforms
+
+
+class ResampleSeriesInputSpec(TraitedSpec):
+    in_file = File(mandatory=True, desc="3D or 4D image file to resample")
```

Comment:
not sure if this will overflow the line length
```suggestion
    in_file = File(exists=True, mandatory=True, desc="3D or 4D image file to resample")
```